hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -489,7 +488,13 @@ private static MapEvictionPolicy findMatchingMapEvictionPolicy(EvictionPolicy ev
      * @return custom eviction policy or {@code null}
      */
     public MapEvictionPolicy getMapEvictionPolicy() {
```

Comment:
can we move this picking right eviction policy logic to [MapContainer#initEvictor](https://github.com/hazelcast/hazelcast/blob/c4be8465df8fc2ae6b78f0feae66c196eb03d84c/hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java#L165) by only having plain getter/setters here? Like this:

```
   public EvictionPolicy getEvictionPolicy() {
        return evictionPolicy;
    }

    public MapConfig setEvictionPolicy(EvictionPolicy evictionPolicy) {
        this.evictionPolicy = checkNotNull(evictionPolicy, "evictionPolicy cannot be null");
        return this;
    }

    public MapEvictionPolicy getMapEvictionPolicy() {
        return mapEvictionPolicy;
    }
   
    public MapConfig setMapEvictionPolicy(MapEvictionPolicy mapEvictionPolicy) {
        this.mapEvictionPolicy = checkNotNull(mapEvictionPolicy, "mapEvictionPolicy cannot be null");
        return this;
    }

Reply:
done